jdbc-driver/src/main/java/com/google/cloud/dataproc/DataprocDriver.java

```diff
@@ -0,0 +1,59 @@
+package com.google.cloud.dataproc;
+
+import com.google.common.annotations.VisibleForTesting;
+import java.sql.*;
```

Comment:
In general, avoid using wildcard imports. I think the `google-java-format` linter will mention that as well.